torchinductor/lowering.py

```diff
@@ -490,6 +490,22 @@ def squeeze_(x, dim=None):
     return x
 
 
+@register_lowering(aten.ceil)
+def ceil(x):
+    if is_integer_type(x):
+        return x
+    fn = ops_wrapper(aten.ceil.__name__)
```

Comment:
we already have ops.floor and ops.ciel, would it suffice to use those instead?

```
@make_pointwise
def ceil(x):
    return ops.ceil(x)
```

Reply:
tl.libdevice.ceil doesn't support int types 

we have this dtype specific lower to avoid calling op.ceil for int type.

Reply:
ah ok, that lgtm then, let @ngimel take a pass at it too